Makefile

```diff
@@ -46,7 +46,7 @@ pgg-mode:	source/native/symbulation_pgg.cc
 	$(CXX_nat) $(CFLAGS_nat) source/native/symbulation_pgg.cc -o symbulation_pgg
 
 sgp-mode:	source/native/symbulation_sgp.cc
-	$(CXX_nat) $(CFLAGS_nat) source/native/symbulation_sgp.cc -o symbulation_sgp
+	$(CXX_nat) $(CFLAGS_nat) source/native/symbulation_sgp.cc -o symbulation_sgp -fsanitize=address
```

Comment:
You should revert this back so that people don't forget to remove it and get stuck with long compile times when it isn't necessary. Remember to git add specific files instead of * so that you don't put in things that you don't mean to be changed